exercises/rna-transcription/bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "rna-transcription",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-console": "^3.0.0",
```

Comment:
I don't think you need all of these.

Reply:
Right, not sure what the deal is with the individual bower json files, the README isn't so clear. All of the exercises seem to have the same set of dependencies

Reply:
I believe all the exercises share the same deps now so that we can cache easier on CI

Reply:
Oh....

Ok, ignore me then.

Reply:
@houli Can you open an issue with details about what was unclear in the README, so I submit a PR to clear things up for future contributors?

@paf31 Yes, recently changed, details in: https://github.com/exercism/purescript/pull/71, https://github.com/exercism/purescript/pull/72 and https://github.com/exercism/purescript/pull/73